tests/__app/generated/nexus-plugin-prisma-typegen.d.ts

```diff
@@ -0,0 +1,165 @@
+import * as Typegen from '../../../src/typegen/static'
```

Comment:
We need these generated files for now because tests in CI fail (ts, type errors, static). We can re-think how we manage our tests suites later if we want. Mainly to avoid having to commit these derived files. But until then these are needed.